src/SFA.DAS.Reservations.Application/Reservations/Commands/CacheReservationEmployer/CacheReservationEmployerCommandValidator.cs

```diff
@@ -35,12 +35,19 @@ public async Task<ValidationResult> ValidateAsync(CacheReservationEmployerComman
             }
             else
             {
-                var globalRules = await _rulesService.GetAccountFundingRules(command.AccountId);
-                if (globalRules.GlobalRules.Any(c => c != null && c.RuleType == GlobalRuleType.ReservationLimit) &&
-                    globalRules.GlobalRules.Count(c => c.RuleType == GlobalRuleType.ReservationLimit) > 0)
+                var accountFundingRulesApiResponse = await _rulesService.GetAccountFundingRules(command.AccountId);
+                if (accountFundingRulesApiResponse.GlobalRules.Any(c => c != null && c.RuleType == GlobalRuleType.ReservationLimit) &&
+                    accountFundingRulesApiResponse.GlobalRules.Count(c => c.RuleType == GlobalRuleType.ReservationLimit) > 0)
```

Comment:
Isn't the bottom check asking the same thing as the top one?

Top asks if there are any global rules that are not null and have a type of reservation limit
bottom one asks for the number of rules with the same type and then said is there more than zero.

Reply:
they are different, one is checking global rules, the other account rules